CMakeLists.txt

```diff
@@ -1,19 +1,12 @@
 cmake_minimum_required(VERSION 3.5)
 
 project(pplib
-    VERSION 1.0.5
     LANGUAGES C)
+set(VERSION 1.0.5)
```

Comment:
Why is this change needed?

Reply:
That's because the CMake support of zlib uses `set(VERSION ...)` which for some reason conflicts with `project(... VERSION ...)`.